### ISSUE TITLE
Show name of team that PR requests review from

### ISSUE THREAD

**Issue by stembrain** [ptsochantaris/trailer#379]
Trailer is so awesome. I have a small request, I don't know if its an API limitation.

If I tag individuals in my org as reviewers on a PR, Trailer displays whether or not they've approved etc:
![image](https://user-images.githubusercontent.com/1630902/52063471-bcf7c580-2540-11e9-9e5d-889a3b9c6e53.png)

But if I tag a team, trailer displays nothing. This causes me to ask myself "Did I forget to ask for a review, or did I request a review from a team and Trailer isnt showing me that?"

Because our org handles pull requests by requesting reviews from people and teams, its important to tag someone. If I dont tag anyone, theres a very low chance anyone would look at the PR, since we handle many per day. 

It would be great if trailer showed me the teams tagged for review on the PR.

**Comment by stembrain** [comment]
This seems different than https://github.com/ptsochantaris/trailer/issues/277

**Comment by ptsochantaris** [comment]
Hi @stembrain and sorry for my very late response. You're right, this is separate from that issue. I'll need to investigate the API to tell you more about whether it's possible to make this tweak, and time has been a bit limited recently as you can tell. However, I'll try to look into this soon and let you know. If you are able to see what the API returns in such cases, it would definitely help speed up my response to this :)

**Comment by ptsochantaris** [comment]
Hi and happy new year! I've been doing some Trailer work recently, and currently have a beta for the upcoming release which includes this feature. If you feel like taking a look at it and doing some testing as well, you can find it here: https://github.com/ptsochantaris/trailer/pull/400

**Comment by stembrain** [comment]
Hi Paul,

I tried the download out; it asked me to update one build version, which I
did. So this testing happened on 1.7.0 build 1530

 I noticed a few things right off the bat- if I Test the API endpoint for
my github enterprise server, the test fails.

Also, where I used to have just 3 pull requests displayed, I now see 100.
It looks like maybe the Pull Requests dropdown in the Repositories tab,
which I have set to 'Participated', isn't being respected. Maybe it's in
some weird limbo state because it wasn't able to talk to the v3 API.

I downgraded back to 1.6.17 build 1497, and the API calls work again, and I
only see the 3 pull requests.


Thanks so much!

-David

**Comment by ptsochantaris** [comment]
Hi @stembrain I thought I'd ping to to see if you are interested in giving the beta another go, there have been a few tweaks related to GE servers so this may help solve your issue above.

**Comment by ptsochantaris** [comment]
Hi, I'm closing this issue for now, but please feel free to reopen if you are still experiencing problems!

**Comment by stembrain** [comment]
Hi @ptsochantaris , I gave 1.7.x a try today, and had some difficulties with it. It seemed to lose my 1.6.17 settings, and was also unable to connect to the v3 github enterprise server that I use. I was able to find the Trailer settings I'd exported years ago, but they didn't properly import to 1.7.x either. 

I downgraded back to 1.6.17 and imported the settings from circa 2017. That brought most of the settings back, but not all of them. I was able to recover them all :) 

In short, I wasn't able to get v3 or v4 API to work with the latest stable release. I'm happy to keep helping you test as I'm able, and to support you with beverage tips :)
